Makefile

```diff
@@ -112,9 +112,12 @@ agent:
 
 ui:
 	cd ui/ && docker build \
-    		  --tag=$(DOCKERHUB_REPO)/$(DOCKER_IMAGE_NAME_PREFIX)-ui:$(REF_TAG) \
-    		  --tag=$(DOCKERHUB_REPO)/$(DOCKER_IMAGE_NAME_PREFIX)-ui:$(ORB_VERSION) \
-    		  --tag=$(DOCKERHUB_REPO)/$(DOCKER_IMAGE_NAME_PREFIX)-ui:$(ORB_VERSION)-$(COMMIT_HASH) \
-    		  -f docker/Dockerfile .
+		--build-arg ENV_PS_SID=${PS_SID} \
+		--build-arg	ENV_PS_GROUP_KEY=${PS_GROUP_KEY} \
```

Comment:
please fix this white space

Reply:
@weyrick I think this request must be resolved by you prior to merging.
I left a comment in the PR as well on current behavior.